### PR TITLE
ADI creation

### PR DESCRIPTION
**Pull request by ibrechin**
Implements the generation of ADI reconciliation files for both valid payments and refunds.

### CODE REVIEW

**Comment by ibrechin** [comment]
@marcofucci please review

File: mtp_bank_admin/apps/bank_admin/adi.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        if not style:
+            style = defaultdict(dict)
+            style.update(config.ADI_JOURNAL_FIELDS[field]['style'])
+
+        if extra_style:
+            for key in extra_style:
+                style[key].update(extra_style[key])
+
+        for key in style:
+            self.journal_ws[cell].__setattr__(
```
Comment (marcofucci) [inline mtp_bank_admin/apps/bank_admin/adi.py]: wouldn't it be cleaner to use `setattr(obj, name, value)` instead of `__setattr__`?

**Comment by marcofucci** [comment]
@ibrechin if you change `__setattr__` to `setattr()` I'll merge it :-)

**Comment by ibrechin** [comment]
@marcofucci done. Although also considering squashing the first 5 commits...

**Comment by marcofucci** [comment]
@ibrechin :+1: I'll wait then

**Comment by marcofucci** [comment]
retest this please

**Pull request merged**